lib/util/config.js

```diff
@@ -59,6 +59,8 @@ var defaults = {
 	expiredBufferMS: 30000,
 	staleBufferMS: 180000,
 	appAuth: undefined,
+	webhooks: undefined,
```

Comment:
It doesn't look like the webhooks config will ultimately be used from here; do we really need to add it?